bindings/go/evmc/host_test.go

```diff
@@ -51,18 +49,18 @@ func (host *testHostContext) GetTxContext() TxContext {
 	return txContext
 }
 
-func (host *testHostContext) GetBlockHash(number int64) common.Hash {
-	return common.Hash{}
+func (host *testHostContext) GetBlockHash(number int64) Hash {
+	return Hash{}
 }
 
-func (host *testHostContext) EmitLog(addr common.Address, topics []common.Hash, data []byte) {
+func (host *testHostContext) EmitLog(addr Address, topics []Hash, data []byte) {
 }
 
 func (host *testHostContext) Call(kind CallKind,
-	destination common.Address, sender common.Address, value *big.Int, input []byte, gas int64, depth int,
-	static bool, salt *big.Int) (output []byte, gasLeft int64, createAddr common.Address, err error) {
+	destination Address, sender Address, value *big.Int, input []byte, gas int64, depth int,
```

Comment:
As we are changing types anyway, we may change `value *big.Int` to `value Hash` or better to `value Uint256BE` where `type Uint256BE = Hash`.